### DISCUSSION TITLE
Unable to access meta info defined in `definePage` at plugin's `extendRoute`

### DISCUSSION THREAD

**Discussion by carrypann** [posva/unplugin-vue-router#392] [Questions and Help] [answered]
Unable to access meta info defined in `definePage` at plugin's `extendRoute`, but the meta info defined in the `<route></route>` block can be accessed.  The following is the sample code：
```
VueRouter({
  extendRoute: (route) => {
    console.log(route.node.value.overrides.meta) //  print undefined even if meta is defined in `definePage`
  }
})
```
Is this a bug or designed this way?

**Comment by posva** [comment, marked as answer]
It's just `route.meta` 😅

**Reply by mkeyy0** [reply]
Hi, @posva. I have a similar issue but in my case `route.meta` is an empty object. `meta` is visible in runtime but not in the extendRoute method and meta is always empty inside it. Please see [this link](https://stackblitz.com/edit/vitejs-vite-tgjnsjna?file=vite.config.js), there's a small reproduction.

**Reply by posva** [reply]
It's because the runtime meta defined with `definePage()` is a runtime js, so it won't appear in the vite (build) plugin
